practice/ttt/test/tic_tac_toe_test.clj

```diff
@@ -0,0 +1,60 @@
+(ns tic-tac-toe-test
+  (:require
+    [clojure.set]
+    [clojure.test :refer :all]
+    [ttt.tic-tac-toe :refer :all]))
+
+
+(deftest winner-of-collection-test
+  (testing "With all :e elements"
+    (is (= nil (winner-of-collection [:e :e :e]))))
+
+  (testing "With all :x elements "
+    (is (= :x (winner-of-collection [:x :x :x]))))
+
+  (testing "With 2 :x elements"
+    (is (= nil (winner-of-collection [:x :x :o])))))
+
+
+(deftest valid-board?-test
```

Comment:
can change this to `test-valid-board?`, having `?` in the middle of a symbol is slightly unorthodox.

Reply:
I agree. Named it this way because consistency and also according to the styleguide :  

> ```
> ;; good
> (deftest something-test ...)
> 
> ;; bad
> (deftest something-tests ...)
> (deftest test-something ...)
> (deftest something ...)
> ```